images/dockerregistry/Dockerfile

```diff
@@ -9,6 +9,9 @@ FROM openshift/origin-base
 RUN yum install -y tree findutils epel-release && \
     yum clean all
 
+# The registry doesn't require a privileged user.
+USER 1001
```

Comment:
Ok, will move.  It's not a big issue, it can run with a higher or lower SCC.  If users would like to run with this particular SCC the workaround is just to set a UID in the dc.  Not a show stopper.